src/main/resources/db/migration/V1_3__create_tables.sql

```diff
@@ -6,32 +10,35 @@ CREATE TABLE VURDERING
     created_at         timestamptz NOT NULL,
     updated_at         timestamptz NOT NULL,
     veilederident      VARCHAR(7)  NOT NULL,
-    type               VARCHAR(20) NOT NULL,
-    begrunnelse        TEXT
+    type               TEXT NOT NULL,
```

Comment:
Er det sånn at vi egentlig bare kan bruke `TEXT` på alle strenger så lenge vi ikke vet den alltid vil ha en bestemt lengde?

Reply:
Vi diskuterte det litt, om hvorfor bruke VARCHAR hvis man ikke vet at det finnes en maksgrense, da er TEXT kanskje like greit 🤔 Vi så for oss at plutselig kan vi få en type over 20 chars, så da endret vi like godt til TEXT. Ulempen er vel som Geir sa at man kan sende inn noe kjempesvært der, men det begrenses kanskje på andre måter i løsningene våre🤷🏼‍♂️

Reply:
Virker ikke som det er noe ytelsesgevinster ved å begrense seg så like greit med `TEXT` da, så begrenser vi det i koden :+1: